tests/http.projections.js

```diff
@@ -83,11 +89,24 @@ describe('Projections', () => {
 			this.timeout(10 * 1000);
 			const client = new EventStore.HTTPClient(httpConfig);
 
-			const response = await client.projections.assert(assertionProjection, assertionProjectionContent, 'onetime', true, true, true);
+			const response = await client.projections.assert(assertionProjection, assertionProjectionContent, 'onetime', true, true, true, true);
 			await sleep(2000);
 			assert.equal(response.name, assertionProjection);
 		});
 
+		it('Should update projection', async function () {
+			this.timeout(10 * 1000);
+			const client = new EventStore.HTTPClient(httpConfig);
+
+			await client.projections.assert(assertionProjection, assertionProjectionContent, 'onetime', true, true, true);
+			const responseWithEmitEmittedStreamsEnabled = await client.projections.getInfo(assertionProjection);
+			assert.equal(responseWithEmitEmittedStreamsEnabled.config.trackEmittedStreams, true);
```

Comment:
Should be `false` if we remove the above test.
```suggestion
			assert.equal(responseWithEmitEmittedStreamsEnabled.config.trackEmittedStreams, false);
```